ddbstream-lambda-sfn-cdk-ts/README.md

```diff
@@ -0,0 +1,161 @@
+
+# Amazon DynamoDB Stream to AWS Step Functions Trigger
+
+This Pattern demonstrates how to automatically trigger AWS Step Functions workflows in response to changes in DynamoDB tables. `DynamoWorkflowTrigger` lets you connect DynamoDB and Step Functions by allowing you to define event handlers that monitor specific changes in your DynamoDB tables and trigger workflows in response. It leverages Lambda functions to evaluate conditions and start Step Functions state machines with inputs derived from the DynamoDB events.
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/{}
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Node and NPM](https://nodejs.org/en/download/) installed
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Cloud Development Kit](https://docs.aws.amazon.com/cdk/latest/guide/cli.html) (AWS CDK) installed
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ```
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd ddbstream-lambda-sfn-cdk-ts
+    ```
+3. (Optional) Update the environment settings in `app.ts`  if you know exactly what Account and Region you want to deploy the stack to.
+    ```typescript
+    env: { 
+        account: 'YOUR_ACCOUNT_NUMBER', // Replace with your AWS account number
+        region: 'YOUR_REGION'          // Replace with your desired region
+    }
+    ```
+4. To deploy from the command line use the following:
+    ```bash
+      npm install
+      npx cdk bootstrap aws://accountnumber/region
```

Comment:
can be removed, CDK is prerequisite for setup, we can expect this to be done already